bot/interact_test.go

```diff
@@ -0,0 +1,76 @@
+package bot
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func requireHasError(t *testing.T, errorMessage string) {
+	require.NotEmpty(t, errorMessage)
+
+	if len([]rune(errorMessage)) > 150 {
+		t.Errorf("error message is too long: %s", errorMessage)
+	}
+
+}
```

Comment:
helper function that validates an error message:

1. is present
2. is shorter than 150 chars (see #71)